CHANGELOG.md

```diff
@@ -6,6 +6,97 @@ This file is generated automatically. Add individual changelog entries to the 'c
 
 This changelog documents all notable changes to VAST and is updated on every release. Changes made since the last release are in the [changelog/unreleased directory][unreleased].
 
+## [v2.0.0-rc1]
+
+### :zap: Breaking Changes
+
+- We removed the experimental `vast get` command. It relied on an internal unique event ID that was never exposed to the user except for debug messages. This removal is a preparatory step towards a simplification of some of the internal workings of VAST.
+  [#2121](https://github.com/tenzir/vast/pull/2121)
+
+- The `meta-index` is now called the `catalog`. This affects multiple metrics and entries in the output of `vast status`, and the configuration option `vast.meta-index-fp-rate`, which is now called `vast.catalog-fp-rate`.
+  [#2128](https://github.com/tenzir/vast/pull/2128)
+
+- The command line option `--verbosity` has the new name `--console-verbosity`. This synchronizes the CLI interface with the configuration file that solely understands the option `vast.console-verbosity`.
+  [#2178](https://github.com/tenzir/vast/pull/2178)
+
+### :warning: Changes
+
+- Terraform scripts to deploy VAST on AWS Lambda and Fargate.
+  [#2108](https://github.com/tenzir/vast/pull/2108)
+
+- We revised the query scheduling logic to exploit synergies when multiple queries run at the same time. In that vein, we updated the related metrics with more accurate names to reflect the new mechanism. The new keys `scheduler.partition.materializations`, `scheduler.partition.schedulings`, and `scheduler.partition.lookups`  issue periodic counts of partitions loaded from disk and scheduled for lookup, and the overall number of queries issued to partitions respectively. The keys `query.workers.idle`, and `query.workers.busy` were renamed to `scheduler.partition.remaining-capacity`, and `scheduler.partition.current-lookups`. Finally, the key `scheduler.partition.pending` counts the number of currently pending partitions. It is still possible to opt-out of the new scheduling algorithm with the option `--use-legacy-query-scheduler`, but that is immediately deprecated and will not be included in the next minor release.
```

Comment:
`s/schedulings/scheduled`, we changed that late on in the PR but forgot to update the changelog entry.